### PR TITLE
[WEEX-86][doc] Update image component's document.

### PR DESCRIPTION
**Pull request by cxfeng1**
Add more examples(basic, placeholder, base64, resize, etc.) and add more details for usage notes.

issue: 86

### CODE REVIEW

File: source/references/components/image.md [hunk 1]
```diff
@@ -6,122 +6,146 @@ order: 8.02
...
+
+### `load`
+
+`load` event handler will be called when the image specified by `src` is loaded. 
+
+**Event object**:
+
+- `success`: {Boolean} It shows that whether the image is loaded successfully.
+- `size`: {Object} The loaded image size
+  - `naturalWidth`: {Number} The intrinsic width of image displayed on device,  it will be zero if the specified image cannot be loaded correctly.
```
Comment (jondong) [inline source/references/components/image.md]: space

File: source/references/components/image.md [hunk 2]
```diff
@@ -6,122 +6,146 @@ order: 8.02
...
+    // TODO
+  } else {
+    console.log(result.errorDesc)
   }
-  .paragraph{
-    margin-bottom: 15px;
+})
+```
+
+See the [complete example](http://dotwe.org/vue/fadcd44a7031943ff0feaaf1895df414).
```
Comment (jondong) [inline source/references/components/image.md]: Complete example goes [here](http://dotwe.org/vue/fadcd44a7031943ff0feaaf1895df414).

File: source/references/components/image.md [hunk 3]
```diff
@@ -6,122 +6,146 @@ order: 8.02
...
+        // TODO
+      }
+    }
   }
-</style>
+}
 ```
 
-[try it](http://dotwe.org/vue/e2122bc245beafb0348d79bfd1274904)
+See the [complete example](http://dotwe.org/vue/94de9307517240dec066d2ea57fe54a0).
```
Comment (jondong) [inline source/references/components/image.md]: Complete example goes [here](http://dotwe.org/vue/94de9307517240dec066d2ea57fe54a0)

File: source/cn/references/components/image.md [hunk 4]
```diff
@@ -6,181 +6,148 @@ order: 8.02
...
+### `save` <span class="api-version">v0.16.0+</span>
 
-## 事件
+保存图片内容到本地文件或相册，此操作可能需要设备相关权限。
+
+**参数**:
+
+- `callback`: {Function} 在图片被写入到本地文件或相册后调用的函数。
+  - `success`: {Boolean} 标记图片是否已写入完成。
+  - `errorDesc`: {String} 如果图像没有成功写入，该字符串包含了详细的错误描述。
```
[the remark this answers is not in the document]
Comment (CYJB) [reply source/cn/references/components/image.md]: I think `success` and `errorDesc` means parameter of callback function? Indent by spaces can make sub-list in markdown.
It is better to point out `success` and `errorDesc` is parameter, and use ordered list

[the remark this answers is not in the document]
Comment (cxfeng1) [reply source/cn/references/components/image.md]: @CYJB that's a good idea😆

[the remark this answers is not in the document]
Comment (jondong) [reply source/cn/references/components/image.md]: ah, make sense. 👍

File: source/cn/references/components/image.md [hunk 5]
```diff
@@ -6,181 +6,148 @@ order: 8.02
...
-  查看 [组件通用样式](../common-style.html)
+### `save` <span class="api-version">v0.16.0+</span>
 
-## 事件
+保存图片内容到本地文件或相册，此操作可能需要设备相关权限。
+
+**参数**:
+
+- `callback`: {Function} 在图片被写入到本地文件或相册后调用的函数。
+  - `success`: {Boolean} 标记图片是否已写入完成。
```
[the remark this answers is not in the document]
Comment (cxfeng1) [reply source/cn/references/components/image.md]: Actually, the spaces here is used to make a sub-list to identify that `success` and `errorDesc` are parameters of the `callback`.
@CYJB has given a good advice, to point out the parameters more clearly.

**Pull request merged**